src/features/orgs/OrgSettingsPage.tsx

```diff
@@ -75,8 +92,16 @@ export const OrgSettingsPage = () => {
 
   const onSubmit: SubmitHandler<OrgAdminFormSchema> = async data => {
     try {
-      await updateOrg(orgId, data);
+      const d = {
+        ...data,
+        // these need to be null to clear them
+        guild_id: guildInfo?.id ?? null,
+        guild_role_id: data.guild_role_id ?? null,
+      };
+      await updateOrg(orgId, d);
       refetch();
+      await fetchManifest();
```

Comment:
@levity this is an attempt to get the useLoginData stuff to reload for the `AddPage` but it isn't working - need some help here.